t/Hydra/Plugin/GithubStatus/GithubStatus.t

```diff
@@ -123,4 +123,48 @@ subtest "extractGithubArgsFromFlake" => sub {
     };
 };
 
+subtest "extractGithubArgsFromInput" => sub {
+    my @failingURIs = (
+        '',
+        'a static string without slashes'
+    );
+    for my $uri (@failingURIs) {
+        my $githubArgs = Hydra::Plugin::GithubStatus::extractGithubArgsFromInput($uri, "the-revision");
+        is($githubArgs, undef, "$uri is not a parsable GitHub URI");
```

Comment:
Same here: to make the empty string more noticeable in the output:

```suggestion
        is($githubArgs, undef, "uri '$uri' is not a parsable GitHub URI");
```

Also, maybe this should be `ok(!defined($githubArgs), ....)` as earlier? IIRC, comparing to `undef` and using `defined()` have slightly different semantics.